### PR TITLE
Upgrade/600/storage volume temlates

### PR DESCRIPTION
**Pull request by sijeesh**
Storage Volume Template :: Support for OneView API version 600

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/storage_volume_templates.py [hunk 1]
```diff
@@ -41,12 +41,11 @@
 
 options = {
     "name": "OneView Test Volume Template",
-    "provisioning":
-        {
-            "capacity": "235834383322",
-            "provisionType": "Thin",
-            "storagePoolUri": ""
-        }
+    "provisioning": {
```
Comment (soodpr) [inline examples/storage_volume_templates.py]: i checked the spec, and there its provisioningType not provisioning. Is it a doc defect?

File: tests/unit/resources/storage/test_storage_volume_templates.py [hunk 2]
```diff
@@ -102,7 +102,7 @@ def test_get_connectable_volume_templates_called_once(self, mock_get):
 
     @mock.patch.object(ResourceClient, 'get')
     def test_get_reachable_volume_templates_called_once(self, mock_get):
-        uri = '/rest/storage-volume-templates/reachable-volume-templates?networks=\'/rest/fake\'&start=0&count=-1'
+        uri = '/rest/storage-volume-templates/reachable-volume-templates?networks=\'/rest/fake\'&start=0&count=-1&privateAllowedOnly=False'
```
Comment (soodpr) [inline tests/unit/resources/storage/test_storage_volume_templates.py]: Do we need to add scopeUris test as well? as we did for other resources.

File: examples/storage_volume_templates.py (hunk 1 above)
Comment (sijeesh) [reply examples/storage_volume_templates.py]: Yes, Its expecting provisioning property and its not documented.

File: tests/unit/resources/storage/test_storage_volume_templates.py (hunk 2 above)
Comment (sijeesh) [reply tests/unit/resources/storage/test_storage_volume_templates.py]: No, its just a get end point.

**Comment by sijeesh** [comment]
Added examples for two endpoints. (missed to add with API version 500 updates)

**Pull request merged**